### PR TITLE
Add Pipeline Substation preset

### PR DESCRIPTION
**Pull request by arch0345**
Adds a preset for [`pipeline=substation`](https://wiki.openstreetmap.org/wiki/Tag:pipeline%3Dsubstation). I noticed that the preset for `pipeline=valve` was under `man_made/pipeline` so I decided to also add this preset here. Would it make more since for these to get moved to its own directory as they don't have `man_made=pipeline`?

### CODE REVIEW

**Comment by tyrasd** [comment]
> Would it make more since for these to get moved to its own directory

Yes, please. All presets in a directory should be sub-presets of the corresponding parent preset (here: `man_made/pipeline`), meaning that they should have the parent's tags (here: `man_made=pipeline`) plus a number of additional tags.

I'd move the Pipeline Valve and Pipeline Substation presets into a new top-level `pipeline` directory. Optimally alongside a generic (unsearchable, low `matchScore`) preset for [other `pipeline=*`](https://taginfo.openstreetmap.org/keys/pipeline#values) objects.

**Comment by arch0345** [comment]
> I'd move the Pipeline Valve and Pipeline Substation presets into a new top-level `pipeline` directory. Optimally alongside a generic (unsearchable, low `matchScore`) preset for [other `pipeline=*`](https://taginfo.openstreetmap.org/keys/pipeline#values) objects.

I noticed other unsearchable generic presets such as the one for `power=*` don't have a specified `matchScore` so I left it out

**Comment by tyrasd** [comment]
Right. Some of these fallback presets do have it (e.g. the one for `man_made=*`), but it's actually redundant since the `"searchable": false` should already be enough to keep it from matching where we don't want it to match.

File: data/presets/pipeline/substation.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "icon": "temaki-powered_pump",
+    "fields": [
+        "name",
+        "substation",
```
Comment (tyrasd) [inline data/presets/pipeline/substation.json:5]: This field has strings for options which are specific for electrical substations. We should probably add a dedicated `substation/pipeline` field for this with the [corresponding values](https://wiki.openstreetmap.org/wiki/Key:substation#Pipeline_dedicated_substations).

**Review by tyrasd — CHANGES_REQUESTED** [review]
Thanks. Looks great. I only noticed one issue with the `substation` field, see below.

File: data/presets/pipeline/substation.json (hunk 1 above)
Comment (arch0345) [reply data/presets/pipeline/substation.json]: Thanks for catching this!

File: data/presets/_pipeline.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "operator"
+    ],
+    "geometry": [
+        "point",
+        "vertex",
+        "line",
+        "area"
+    ],
+    "tags": {
+        "natural": "*"
```
Comment (tyrasd) [inline data/presets/_pipeline.json:14]: I assume this should have been the `pipeline` tag here, right? I've totally overlooked it when reviewing the PR. :see_no_evil:

Comment (tyrasd) [reply data/presets/_pipeline.json:14]: This was spotted in https://www.transifex.com/openstreetmap/id-editor/translate/#pl/$/442611093 by @gdabski

It's fixed in 4f70a708b3a96c02104777f1cfecef895c73970e

**Pull request merged**